terraform/provider.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    aws = {
```

Comment:
@zackads @Ninamma @stuart-madetech See for requiring providers in Terraform: https://www.terraform.io/docs/language/providers/requirements.html#requiring-providers